components/infobox/wikis/starcraft2/infobox_person_player_custom.lua

```diff
@@ -324,34 +314,48 @@ function CustomPlayer._getEarningsMedalsData(player)
 		playerConditions,
 		ConditionNode(ColumnName('date'), Comparator.neq, '1970-01-01 00:00:00'),
 		ConditionNode(ColumnName('liquipediatiertype'), Comparator.neq, 'Charity'),
-		ConditionNode(ColumnName('liquipediatiertype'), Comparator.neq, 'Qualifier'),
 		ConditionTree(BooleanOperator.any):add({
 			ConditionNode(ColumnName('individualprizemoney'), Comparator.gt, '0'),
+			ConditionNode(ColumnName('extradata_award'), Comparator.neq, ''),
 			ConditionTree(BooleanOperator.all):add({
 				ConditionNode(ColumnName('players_type'), Comparator.gt, 'solo'),
 				placementConditions,
 			}),
 		}),
 	})
 
-	local data = CustomPlayer._getLPDBrecursive(conditions:toString(), query, 'placement')
-
 	local earnings = {}
 	local medals = {}
 	earnings['total'] = {}
 	medals['total'] = {}
 	local earnings_total = 0
 
-	if type(data[1]) == 'table' then
-		for _, item in pairs(data) do
-			--handle earnings
-			earnings, earnings_total = CustomPlayer._addPlacementToEarnings(earnings, earnings_total, item)
+	local queryParameters = {
+		conditions = conditions:toString(),
+		order = 'liquipediatier asc, weight desc',
+	}
 
-			--handle medals
-			medals = CustomPlayer._addPlacementToMedals(medals, item)
-		end
+	local itemChecker = function(item)
```

Comment:
```suggestion
	local processTournament = function(item)
```
Or another better name, itemChecker is not descriptive.

Reply:
Update the function parameter name also

Reply:
done